bin/builder.js

```diff
@@ -1,25 +1,22 @@
 #!/usr/bin/env node
 "use strict";
 
-// Configuration
-var Config = require("../lib/config");
-var config = new Config();
+var resolve = require("resolve");
+var log = require("../lib/log");
+var localBuilder;
 
-// Set up environment
-var Environment = require("../lib/environment");
-var env = new Environment({
-  config: config
-});
+// Try to resolve builder in the current project's root, use global if fails
+try {
+  localBuilder = resolve.sync(
```

Comment:
What does `resolve.sync` offer over `require.resolve`? Ideally we don't add a new dependency...